### PR TITLE
section download 

### PR DESCRIPTION
**Pull request by mxuanvan02**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -34,8 +34,55 @@
...
+    </div>
+  </section>
+  <!--================================= Section Download =================================-->
+  <section class="download">
+    <div class="download-content">
+      <p class="download-content-title">Get Your Free Version</p>
+      <p class="download-content-description">Having used discount toner cartridges for twenty years, there have been a lot o changes
+                                      in the toer cartridge market. The market today is approximately a twenty billon dollar business.</p>
+    </div>
+    <div class="download-button"><a>Download</a></div>
```
Comment (nhungduong-agilityio) [inline src/index.html]: I think not need wrap div tag in this line. And it should be a button?

File: src/css/index.css [hunk 2]
```diff
@@ -171,3 +170,158 @@ header {
...
+  display: flex;
+  align-items: center;
+  background: #EBCB80;
+  margin: 0 auto;
+  border-radius: 50px;
+}
+
+
+.download-button a {
+  text-transform: uppercase;
```
Comment (nhungduong-agilityio) [inline src/css/index.css]: I see you use this style multiple time, you should define class for this style to reuse. Ex: .text-uppercase {} --> class="... text-uppercase"

File: src/css/index.css [hunk 3]
```diff
@@ -171,3 +170,158 @@ header {
...
+  margin: 0 auto;
+}
+
+
+.download-button {
+  width: 210px;
+  height: 48px;
+  display: flex;
+  align-items: center;
+  background: #EBCB80;
```
Comment (nhungduong-agilityio) [inline src/css/index.css]: Should add color variable, please check more colors

File: src/css/index.css [hunk 4]
```diff
@@ -171,3 +170,158 @@ header {
...
+  top: 1536px;
+  background: linear-gradient(0deg, #4b4b4b, #4b4b4b), url(../images/download/background.png);
+  background-blend-mode: soft-light, normal;
+}
+
+
+.download-content {
+  text-align: center;
+  color: var(--txt-color-white);
+  font-family: 'Roboto';
```
Comment (nhungduong-agilityio) [inline src/css/index.css]: Should add generic-family? Check all case

File: src/css/index.css [hunk 5]
```diff
@@ -171,3 +170,158 @@ header {
...
+                              #SECTION DOWNLOAD
+\* ---------------------------------------------------------------------- */
+
+
+.download {
+  position: absolute;
+  width: 1440px;
+  height: 680px;
+  left: 0;
+  top: 1536px;
```
Comment (nhungduong-agilityio) [inline src/css/index.css:282]: Please check again, top value too high. Check all same case

Comment (mxuanvan02) [reply src/css/index.css:282]: top value equal header height plus features section height, if not download section is behind header.

Comment (mxuanvan02) [reply src/css/index.css:282]: > Please check again, top value too high. Check all same case

top value equal header height plus features section height, if not download section is behind header.

**Pull request merged**